settings.py

```diff
@@ -2118,6 +2119,32 @@
     # Used in tests.  TODO - use override_settings instead
     'ewsghana-test-input-stock': 'custom.ewsghana',
     'test-pna': 'custom.intrahealth',
+    'abtmali': 'custom.abt',
```

Comment:
Could you add a comment or a newline to separate this from the above two lines which have the header comment ```# Used in tests.  TODO - use override_settings instead```?

Reply:
@nickpell updated

Reply:
Thanks